### ISSUE TITLE
Switch to white icon in yosemite dark mode

### ISSUE THREAD

**Issue by bitsandnumbers** [Leandros/ownDrop#25]
Could you implement a switch to white menubar icon for Yosemite "dark mode", please ?

Edit: here is a link to some explanations on ownCloud repo:
https://github.com/owncloud/client/issues/2215

**Comment by FelixAkk** [comment]
Oh this would indeed be very nice! Very nice app by the way!

**Comment by eppfel** [comment]
Bump. Same here. Great App, but the dark icon is not seen at all. Nearly uninstalled it again, because I did not find it.
Maybe I'm going to have look into it myself and create a pull request, but I have no time right now.

**Comment by eppfel** [comment]
I created a Pull Request to resolve this issue. For now you can use my release: https://github.com/dharma-guardian/ownDrop/releases/tag/0.8.1
I will remove the fork after merge.

**Comment by ovidiulixandru** [comment]
@dharma-guardian There is still a small issue with the modification - it doesn't switch between the two icon versions at runtime, only on startup. There are some utilities which switch automatically between the two modes based on the day time (f.lux for example) and you are left with a blank spot in the menu bar on switch. You must then restart ownDrop to get the correct icon. Any chance for a bugfix?

**Comment by eppfel** [comment]
Yes, I only check for Dark Mode on application start. I figured that out, too, but had no quick fix in mind and not much time to put into it.
I want to fix it, but can't promise anything right now.

**Comment by Leandros** [comment]
Gonna look into it.

On Wednesday, 14 October 2015, Felix notifications@github.com wrote:

> Yes, I only check for Dark Mode on application start. I figured that out,
> too, but had no quick fix in mind and not much time to put into it.
> I want to fix it, but can't promise anything right now.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Leandros/ownDrop/issues/25#issuecomment-147953335.

## 

Mit freundlichen Grüßen, / Yours sincerely,

Arvid Gerstmann ag@arvid-g.de

---

Portfolio: http://arvid-gerstmann.de
Twitter: http://twitter.com/ArvidGerstmann

## Google+: http://google.com/+ArvidGerstmann
